src/main/java/com/example/techiteasybackend/dto/input/CiModuleInputDto.java

```diff
@@ -0,0 +1,13 @@
+package com.example.techiteasybackend.dto.input;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class CiModuleInputDto {
+    private long id;
```

Comment:
Id Is niet heel netjes in een inputdto, je wilt niet de id kunnen aanpassen van een model.